### DISCUSSION TITLE
[Esame 13 Febbraio 2023] Esercizio 3

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#351] [Esami - Bandiera]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/aa050a4d-35cf-4353-a7f0-0cfa8bc2835c)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/31fd0efd-7c92-41ff-95fc-e5d83b5996e6)
